pkg/cmd/dev/cmd.go

```diff
@@ -105,19 +105,22 @@ func runCommand(opts *DevOptions) func(cmd *cobra.Command, args []string) {
 
 		if err := opts.complete(cmd.Context(), args); err != nil {
 			output.Errorf("failed to complete options: %s", err)
+			return
 		}
 		if opts.stop {
 			output.Infoln("Stopping development server...")
 			err := stopDev(cmd.Context(), opts)
 			if err != nil {
-				output.Fatalf("Failed to stop dev server: %s", err)
+				output.Errorf("Failed to stop dev server: %s", err)
+				return
 			}
 			output.Infoln("Development server stopped")
 			return
 		}
 		err := runDev(cmd.Context(), opts)
 		if err != nil {
-			output.Fatalf("Failed to start dev server: %s", err)
+			output.Errorf("Failed to start dev server: %s", err)
+			return
```

Comment:
We might want to still use `os.Exit(1)` in this case instead of `return`. Returning will result in the command exiting as "successful".